apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/tips/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { debounce, get, filter, deburr, lowerCase, includes } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __experimentalInserterMenuExtension as InserterMenuExtension } from '@wordpress/block-editor';
```

Comment:
You should check that `__experimentalInserterMenuExtension` is available and bail out if not. Atomic sites might run older versions of the plugin.